ci/upload_to_owncloud.sh

```diff
@@ -50,6 +54,17 @@ main() {
     remote_path="$1"; shift
     [ -n "$1" ] || { usage; err "Missing local-path"; exit 1; }
     local_path="$1"; shift
+    [ -z "${REMOTE_TEMP_PATH-unset}" ] && { usage; err "Temporary remote path is set but empty!"; exit 1; }
```

Comment:
Just to be clear - I think we should *always* upload to a randomly generated REMOTE_TEMP_PATH. Which means, no need for the option since we'll always use it.